plugin/trino-accumulo/src/main/java/io/trino/plugin/accumulo/index/Indexer.java

```diff
@@ -546,7 +556,9 @@ public String toString()
         {
             return toStringHelper(this)
                     .add("row", new String(row.array(), UTF_8))
+                    .add("rowOffset", row.arrayOffset())
                     .add("family", new String(family.array(), UTF_8))
+                    .add("familyOffset", family.arrayOffset())
```

Comment:
does not belong to this commit

Reply:
i think it is in the right commit, why do you think otherwise?

Reply:
Nevermind, I've read https://errorprone.info/bugpattern/ByteBufferBackingArray and it makes sense to add this here.